app/main.py

```diff
@@ -14,4 +14,5 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = sum(1 for char in phrase if char.lower() == letter.lower())
+    return count
```

Comment:
Consider simplifying this code by using a one-liner without the need for creating a separate variable